app/views/presentations/_form.html.erb

```diff
@@ -1,36 +1,15 @@
-<%= form_for presentation do |f| %>
-  <%= render 'errors' %>
+<%= simple_form_for presentation do |f| %>
 
-  <p>
-    <%= f.label :topic %><br>
-    <%= f.text_field :topic %>
-  </p>
-
-  <p>
-    <%= f.label :description %><br>
-    <%= f.text_field :description %>
-  </p>
-
-  <p>
-    <%= f.label :duration %><br>
-    <%= f.text_field :duration %> Minutes
-  </p>
-
-  <p>
-    <%= f.label :event %><br>
-    <%= f.collection_select(:event_id, Event.upcoming, :id, :name, prompt: true) %>
-  </p>
+  <%= f.input :topic %>
+  <%= f.input :description, as: :string %>
```

Comment:
do you need the as here? I would have expected simple form to have figured that on on its own